src/csv/report.rs

```diff
@@ -413,11 +419,13 @@ fn num_plot(table: &[HashMap<String, String>], column: String) -> Vec<BinnedPlot
     plot_data
 }
 
-fn nominal_plot(table: &[HashMap<String, String>], column: String) -> Vec<PlotRecord> {
+fn nominal_plot(table: &[HashMap<String, String>], column: String) -> Option<Vec<PlotRecord>> {
     let mut values = Vec::new();
     for row in table {
         let val = row.get(&column).unwrap();
-        values.push(val.to_owned());
+        if !val.is_empty() {
+            values.push(val.to_owned());
+        }
```

Comment:
Here as well, use map/collect instead of for loop.

Reply:
Done in https://github.com/rust-bio/rust-bio-tools/pull/139/commits/ac61316a59982a6185c7c44adf4632b3708ed41d.